bazel/pypi_repos.bzl

```
@@ -0,0 +1,14 @@
+"""Defines a local PyPI repository for the pyprotoc_plugin, the targets
+should be installed later in WORKSPACE file, to be accessible.
+`pip_parse` takes a `requirements_lock` file as input, which is
+generated by `bazel run :requirements.update` command."""
```

Some wording suggestions:

```suggestion
"""Defines local PyPI-based repositories. Their deps
should be installed later in WORKSPACE file to be accessible.
`pip_parse` takes a `requirements_lock` file as input, which is
generated by the `bazel run :requirements.update` command."""
```